aiosmtpd/controller.py

```diff
@@ -59,16 +59,25 @@ def __init__(
         self.handler = handler
         self.hostname = "::1" if hostname is None else hostname
         self.port = port
-        self.enable_SMTPUTF8 = enable_SMTPUTF8
         self.ssl_context = ssl_context
         self.loop = asyncio.new_event_loop() if loop is None else loop
         self.ready_timeout = os.getenv("AIOSMTPD_CONTROLLER_TIMEOUT", ready_timeout)
         self.server_kwargs: Dict[str, Any] = server_kwargs or {}
+        if enable_SMTPUTF8 is not None:
+            # Uncomment next lines when we hit 1.3
+            # warn("enable_SMTPUTF8 will be removed in the future. "
+            #      "Please use server_kwargs instead.", DeprecationWarning)
+            self.server_kwargs["enable_SMTPUTF8"] = enable_SMTPUTF8
+        else:
+            # This line emulates previous behavior of defaulting enable_SMTPUTF8 to
+            # True. Which actually kinda conflicts with SMTP class's default, but it's
+            # explained in the documentation.
+            self.server_kwargs.setdefault("enable_SMTPUTF8", True)
```

Comment:
I need to implement this code because the docs -- pulled from #202 -- specified this behavior (i.e., `enable_SMTPUTF8` gets injected into `server_kwargs`).